core/src/main/java/com/graphhopper/util/shapes/Polygon.java

```diff
@@ -69,6 +68,11 @@ public Polygon(double[] lat, double[] lon) {
             }
         }
 
+        minLat -= GROWN_FACTOR;
+        minLon -= GROWN_FACTOR;
+        maxLat += GROWN_FACTOR;
+        maxLon += GROWN_FACTOR;
```

Comment:
Here it would be better if we can keep the default constructor and create a second one like

public Polygon(double[] lat, double[] lon, double growFactor)